cfg/cfg.go

```diff
@@ -148,6 +173,11 @@ func Print() {
 	}
 	if IdleTimeout > 0 {
 		logrus.Infof("==> Idle timeout: %s", IdleTimeout)
+	}
+	if RunTimeout > 0 {
+		logrus.Infof("==> Run timeout: %s", RunTimeout)
+	}
+	if IdleTimeout > 0 || RunTimeout > 0 {
 		logrus.Infof("==> Stop retry period: %s", StopRetryPeriod)
 	}
```

Comment:
StopRetryPeriod is related because it is the duration we wait until trying to stop a devworkspace that have failed to stop previously: https://github.com/eclipse-che/che-machine-exec/blob/aa76e6080e188822ab9c60abce6c01791befc3c7/timeout/inactivity.go#L104

Here the the StopRetryPeriod is being printed if timeouts (ie. `IdleTimeout`, `RunTimeout`) are set meaning that che-machine-exec may have to stop the workspace at some point